exercises/pangram/src/example.h

```diff
@@ -0,0 +1,7 @@
+#ifndef _PANGRAM_H
+#define _PANGRAM_H
+
+#include <stdbool.h>
+
+bool is_pangram(const char *sentence);
+#endif
```

Comment:
Can you add a line of whitespace above to be consistent with the formatting of other exercises?